app/case-management/case-management.js

```diff
@@ -52,10 +52,10 @@ const caseDataDao = {
             } else if (params.dueForVl == 0) {
                 where = where + "and " + getDueForVl() + " = 0 "
             }
-            if (params.caseManagerUuid != null) {
-                where = where + "and case_manager_uuid = '" + params.caseManagerUuid + "'"
+            if (params.caseManagerUserId != null) {
+                where = where + "and case_manager_user_id in (" + stripEndQuotes(params.caseManagerUserId) + ")"
```

Comment:
logic to convert params to the right data types and clean it, belongs to the controller, route object. I would move the stripping of quotes out of this service.